tests/unit/python/foglamp/services/core/api/certs/foglamp.cert

```diff
@@ -0,0 +1,15 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
@izoratti what is your take on having keys in the repo for the purpose of testing?

Reply:
@izoratti  please confirm.